examples/webgl_lines_fat.html

```diff
@@ -0,0 +1,288 @@
+
+					var point = spline.getPoint( i / l );
+					positions.push( point.x, point.y, point.z );
+
+					color.setHSL( i / l, 1.0, 0.5 );
+					colors.push( color.r, color.g, color.b );
+
+				}
+
+
+				// THREE.Line2 ( LineGeometry, LineMaterial )
+
+				var geometry = new THREE.LineGeometry();
+				geometry.setPositions( positions );
+				geometry.setColors( colors );
+
+				var material = new THREE.LineMaterial( {
+
+					color: 0xffffff,
+					linewidth: 10, // in pixels
+					vertexColors: THREE.VertexColors,
+					//resolution:  // to be set by renderer, eventually
+
+				} );
+
+				line = new THREE.Line2( geometry, material );
+				line.scale.set( 1, 1, 1 );
+				scene.add( line );
+
+
+				// THREE.Line ( BufferGeometry, LineBasicMaterial ) - rendered with gl.LINE_STRIP
+
+				var geo = new THREE.BufferGeometry();
+				geo.addAttribute( 'position', new THREE.Float32BufferAttribute( positions, 3 ) );
+				geo.addAttribute( 'color', new THREE.Float32BufferAttribute( colors, 3 ) );
+
+				var mat = new THREE.LineBasicMaterial( { vertexColors: THREE.VertexColors } );
+
+				line1 = new THREE.Line( geo, mat );
+				line1.visible = false;
+				scene.add( line1 );
+
+
+				// THREE.Wireframe ( WireframeGeometry2, LineMaterial )
+
+				//var geo = new THREE.BoxBufferGeometry( 16, 16, 4, 2, 2, 1 );
+				//var geo = new THREE.IcosahedronBufferGeometry( 8, 0 );
+				var geo = new THREE.PlaneBufferGeometry( 16, 16, 2, 2 );
+
+				var geometry = new THREE.WireframeGeometry2( geo );
+
+				var material = new THREE.LineMaterial( {
+
+					color: 0x4080ff,
+					linewidth: 10, // in pixels
+					//resolution:  // to be set by renderer, eventually
+
+				} );
+
+				wireframe = new THREE.Wireframe( geometry, material );
+				wireframe.scale.set( 1, 1, 1 );
+				scene.add( wireframe );
+
+
+				// THREE.Line ( WireframeGeometry, LineBasicMaterial ) - rendered with gl.LINE
+
+				geo = new THREE.WireframeGeometry( geo );
+
+				var mat = new THREE.LineBasicMaterial( { color: 0x4080ff } );
+
+				wireframe1 = new THREE.LineSegments( geo, mat );
+				wireframe1.visible = false;
+				scene.add( wireframe1 );
+
+				//
+
+				window.addEventListener( 'resize', onWindowResize, false );
+				onWindowResize();
+
+				stats = new Stats();
+				document.body.appendChild( stats.dom );
+
+				initGui();
+
+			}
+
+			function onWindowResize() {
+
+				camera.aspect = window.innerWidth / window.innerHeight;
+				camera.updateProjectionMatrix();
+
+				renderer.setSize( window.innerWidth, window.innerHeight );
+
+				insetWidth = window.innerWidth / 4;
+				insetHeight = window.innerHeight / 4;
+
+			}
+
+			function animate() {
+
+				requestAnimationFrame( animate );
+
+				stats.update();
+
+				wireframe.geometry.maxInstancedCount = Math.floor( Date.now() / 1000 ) % wireframe.geometry.index.count - 1; // why - 1 needed ?
+
+				// main scene
+
+				// renderer will set this eventually
+				line.material.resolution.set( window.innerWidth, window.innerHeight );
```

Comment:
Noob question. How's that [spite/THREE.MeshLine](https://github.com/spite/THREE.MeshLine) doesn't require this manual step? Are these solutions fundamentally different? Thanks.

Reply:
/ping @spite

Reply:
You also have to specify the resolution on THREE.MeshLine (https://github.com/spite/THREE.MeshLine/blob/master/src/THREE.MeshLine.js#L379), I don't think they're different in that regard.

Reply:
Resolution is required because the line width is in pixels.

Reply:
Thanks guys. I used a line width of 0.003 when using THREE.MeshLine and thought "it just works". It assumes a resolution of 1,1 - now I understand.